docker/entrypoint.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
-mkdir -p  /var/log/go-dmux
+mkdir -p  /var/log/flipkart/go-dmux
+
+#if [[ -z ${LOG_DIR} ]]; then
+#  export LOG_DIR="/var/log/go-dmux"
+#  mkdir -p ${LOG_DIR}
+#fi
 sleep 10
```

Comment:
This is intentional, need to discuss something.

Reply:
Since we will be writing logs to console, we can avoid creating log directory here.
If it's needed, log directory will be created at run time and location will be part of config.

Reply:
I was thinking that since we would have log rotation and might not need older logs, we can keep writing to log files as done today ..